plugins/modules/packaging/os/homebrew.py

```diff
@@ -39,7 +39,7 @@
             - "A C(:) separated list of paths to search for C(brew) executable.
               Since a package (I(formula) in homebrew parlance) location is prefixed relative to the actual path of I(brew) command,
               providing an alternative I(brew) path enables managing different set of packages in an alternative location in the system."
-        default: '/usr/local/bin:/opt/homebrew/bin'
+        default: '/usr/local/bin:/opt/homebrew/bin:/home/linuxbrew/.linuxbrew/bin'
```

Comment:
Why not also do the same modification to the `homebrew_cask` and `homebrew_tap` modules?

Reply:
Done in `homebrew_tap` too. Not done in `homebrew_cask` as Casks are not supported on Linux.